### PR TITLE
[ Fabric ] Add the support of orderer cli in fabric.

### PR DESCRIPTION
**Pull request by lakshyakumar**
Signed-off-by: lakshyakumar <lakshya.kumar@accenture.com>

**Changelog**
- Add the system channel name while creating the genesis file
- Add the orderer_cli in k8_component/templates/orderer_cli.tpl 
- Add the crypto material of admin of orderer org to the vault and delete it on reset.

 

**Reviewed by**
@jagpreetsinghsasan 

 

**Linked issue**
#1072

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/roles/create/genesis/tasks/main.yaml [hunk 1]
```diff
@@ -8,11 +8,7 @@
 - name: "Create genesis block"
   shell: |
     cd {{ build_path }}
-    {% if '2.' in network.version %}
-    ./configtxgen -profile {{ genesis.name }} -channelID syschannel  -outputBlock ./channel-artifacts/genesis.block
-    {% else %}
-    ./configtxgen -profile {{ genesis.name }} -outputBlock ./channel-artifacts/genesis.block
-    {% endif %}
+    ./configtxgen -profile {{ genesis.name }} -channelID syschannel -outputBlock ./channel-artifacts/genesis.block
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/roles/create/genesis/tasks/main.yaml:11]: Is this tested for both 1.4.x and 2.x versions?

File: platforms/hyperledger-fabric/configuration/roles/delete/vault_secrets/tasks/main.yaml [hunk 2]
```diff
@@ -83,6 +83,8 @@
     vault delete secret/crypto/ordererOrganizations/{{ component_name }}/ca
     vault delete secret/crypto/ordererOrganizations/{{ component_name }}/orderers/{{orderer.name}}.{{ component_name }}/tls
     vault delete secret/crypto/ordererOrganizations/{{ component_name }}/orderers/{{orderer.name}}.{{ component_name }}/msp
+    vault delete secret/crypto/ordererOrganizations/{{ component_name }}/users/admin/tls
+    vault delete secret/crypto/ordererOrganizations/{{ component_name }}/users/admin/msp
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/roles/delete/vault_secrets/tasks/main.yaml:87]: Add these to the docs as well
https://blockchain-automation-framework.readthedocs.io/en/latest/architectureref/certificates_path_list_fabric.html

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-fabric/configuration/roles/create/genesis/tasks/main.yaml (hunk 1 above)
Comment (lakshyakumar) [reply platforms/hyperledger-fabric/configuration/roles/create/genesis/tasks/main.yaml:11]: tested for 1.4.4 and it is same for 2.x

File: platforms/hyperledger-fabric/configuration/roles/delete/vault_secrets/tasks/main.yaml (hunk 2 above)
Comment (lakshyakumar) [reply platforms/hyperledger-fabric/configuration/roles/delete/vault_secrets/tasks/main.yaml:87]: added

**Review by jagpreetsinghsasan — APPROVED** [review]

**Pull request merged**